pkg/ansible/controller/reconcile.go

```diff
@@ -285,3 +288,21 @@ func contains(l []string, s string) bool {
 	}
 	return false
 }
+
+// determineReturn - if the object was updated outside of our controller
+// this means that the current reconcilation is over and we should use the
+// latest version. To do this, we just exit without error because the
+// latest version should be queued for update.
+func determineReturn(err error) (bool, error) {
+	exit := false
+	if err == nil {
+		return exit, err
+	}
+	exit = true
+
+	if apierrors.IsConflict(err) {
+		log.V(1).Info("Conflict found during an update, re-run the reconcilation")
```

Comment:
Maybe "Rerunning reconciliation"? As a user, if I were reading this log, I would interpret it as requiring action on my part